langchain/src/chains/base.ts

```diff
@@ -207,6 +207,12 @@ export abstract class BaseChain<
       );
     }
     await runManager?.handleChainEnd(outputValues);
+
+    // If a runnable is being used `outputValues` is
+    // not guaranteed to be an object
```

Comment:
We should guarantee this by only allowing runnables that emit objects? That's why we have static typing

Reply:
I believe we have to. If a user passes in an output parser which returns a string, then the result of `_call` in `LLMChain` will always be a string